packages/ripple/src/runtime/internal/server/index.js

```diff
@@ -85,6 +94,35 @@ export async function render(component) {
 	}
 	return { head, body, css }
 }
+
+/** @type {render} */
+export function renderToStream(component) {
+    const stream = new Readable({
+        read() {}
+    });
+
+    const output = new Output(null, stream);
+
+    (async () => {
```

Comment:
Can we move this out to its own function instead of being an inline anonymous function?